virt/virt-4-8-release-notes.adoc

```diff
@@ -64,6 +64,12 @@ The SVVP Certification applies to:
 
 * {VirtProductName} now supports third-party Container Network Interface (CNI) plug-ins that are link:https://access.redhat.com/articles/5436171[certified by Red Hat] for use with {product-title}.
 
+[id="virt-4-8-quick-starts"]
+=== Quick starts
+
+* Quick start tours are available for several {VirtProductName} features. To view the tours, click the *?* icon in the masthead of the {VirtProductName} console and select *Quick Starts*.
```

Comment:
I wasn't familiar with the term "masthead". Do you think our customers would know where to look? Would it make sense to simplify the location to say "menu bar on the header"?

Reply:
Done